CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing to @dhl-uu/timing-util
+
+## Required software
+
+- Node.js
+- npm
+- git
+- [editorconfig plugin][editorconfig] for your favorite editor
+
+[editorconfig]: https://editorconfig.org/
+
+## First time after cloning the repository
+
+```bash
+npm install
+```
+
+## Code organization
+
+All source code is in ESM format in the `src/` directory. A UMD version of each module is created in the `umd/` directory when we run `npm publish` or manually when we `npm run build`.
+
+Please be mindful of users who need backwards compatibility. Transpiled code can be bulky and polyfills tend to be huge. Don't go wild with ESNext features if you don't really need them.
```

Comment:
I can't say I understand what this means. That's not a problem per se, if the assumed knowledge is also a prerequisite to developing the package - then this just signals to someone of my skill level that they're not ready to start developing. Given the content of the package, that's probably accurate, but if that is not intended, you could include a link to explain what ESNext features are.

Reply:
No, that is not intended at all. Is "ESNext" the only problem or are there others?

Reply:
Personally, these terms are all things I'm inferring from context:
- ESM
- UMD
- transpiled
- polyfills
- ESNext

However, ESNext features is the one thing that you really need to know because it is an actual recommendation.